client/blocks/login/social.jsx

```diff
@@ -140,72 +137,6 @@ class SocialLoginForm extends Component {
 		return `https://${ host + login( { socialService: service } ) }`;
 	};
 
-	renderSocialTos = () => {
-		const { redirectTo, translate } = this.props;
-
-		const isJetpackMagicLinkSignUpFlow =
-			redirectTo &&
-			redirectTo.includes( 'jetpack/connect' ) &&
-			config.isEnabled( 'jetpack/magic-link-signup' );
-
-		const tosLink = (
-			<a
-				href={ localizeUrl( 'https://wordpress.com/tos/' ) }
-				target="_blank"
-				rel="noopener noreferrer"
-			/>
-		);
-		const privacyLink = (
-			<a
-				href={ localizeUrl( 'https://automattic.com/privacy/' ) }
-				target="_blank"
-				rel="noopener noreferrer"
-			/>
-		);
-
-		if ( isJetpackMagicLinkSignUpFlow ) {
-			return (
-				<>
-					<p className="login__social-tos">
-						{ translate(
-							'By continuing, you agree to our {{tosLink}}Terms of' +
-								' Service{{/tosLink}} and acknowledge that you have read our' +
-								' {{privacyLink}}Privacy Policy{{/privacyLink}}.',
-							{
-								components: {
-									tosLink,
-									privacyLink,
-								},
-							}
-						) }
-					</p>
-					<p className="login__social-tos">
-						{ translate(
-							'If you continue with Google, Apple, or an email that isn’t registered yet,' +
-								' you are creating a new WordPress.com account.'
-						) }
-					</p>
-				</>
-			);
-		}
-		return (
-			<p className="login__social-tos">
-				{ translate(
-					'If you continue with Google or Apple,' +
-						' you agree to our' +
-						' {{tosLink}}Terms of Service{{/tosLink}}, and have' +
-						' read our {{privacyLink}}Privacy Policy{{/privacyLink}}.',
-					{
-						components: {
-							tosLink,
-							privacyLink,
-						},
-					}
-				) }
-			</p>
-		);
-	};
-
```

Comment:
Now that we're showing a single ToS in the login page above the "Continue" button, there's no point in having this in the component as well.